features/step_definitions/api_request_steps.rb

```diff
@@ -221,6 +221,17 @@
   }]
 end
 
+# :nocov:
```

Comment:
@MazOneTwoOne have you tried adding wrapping this whole file in the nocov tags?

Reply:
Yeah, I'll be adding this to different parts, to see what happens. It doesn't look promising at the moment